### ISSUE TITLE
Test errors with graphql-core 3.2.0

### ISSUE THREAD

**Issue by mweinelt** [graphql-python/graphql-relay-py#40]
After updating graphql-core to 3.2.0 we're seeing the following tests fail on graphql-relay 3.1.0:

```py
=================================== FAILURES ===================================
_______________________ test_correctly_refetches_rebels ________________________

    @mark.asyncio
    async def test_correctly_refetches_rebels():
        query = """
          query RebelsRefetchQuery {
            node(id: "RmFjdGlvbjox") {
              id
              ... on Faction {
                name
              }
            }
          }
        """
        expected = {
            "node": {"id": "RmFjdGlvbjox", "name": "Alliance to Restore the Republic"}
        }
        result = await graphql(StarWarsSchema, query)
>       assert result == (expected, None)
E       AssertionError: assert ExecutionResult(data={'node': None}, errors=[GraphQLError('Support for returning GraphQLObjectType from resolve_type was removed in GraphQL-core 3.2, please return type name instead.', locations=[SourceLocation(line=3, column=9)], path=['node'])]) == ({'node': {'id': 'RmFjdGlvbjox', 'name': 'Alliance to Restore the Republic'}},\n None)
E         +ExecutionResult(data={'node': None}, errors=[GraphQLError('Support for returning GraphQLObjectType from resolve_type was removed in GraphQL-core 3.2, please return type name instead.', locations=[SourceLocation(line=3, column=9)], path=['node'])])
E         -({'node': {'id': 'RmFjdGlvbjox', 'name': 'Alliance to Restore the Republic'}}, None)
E         Full diff:
E         + ExecutionResult(data={'node': None}, errors=[GraphQLError('Support for returning GraphQLObjectType from resolve_type was removed in GraphQL-core 3.2, please return type name instead.', locations=[SourceLocation(line=3, column=9)], path=['node'])],
E         - (
E         -  {'node': {'id': 'RmFjdGlvbjox',
E         -            'name': 'Alliance to Restore the Republic'}},
E         -  None,
E           )

tests/test_star_wars_object_identification.py:41: AssertionError
_______________________ test_correctly_refetches_empire ________________________

    @mark.asyncio
    async def test_correctly_refetches_empire():
        query = """
          query EmpireRefetchQuery {
            node(id: "RmFjdGlvbjoy") {
              id
              ... on Faction {
                name
              }
            }
          }
        """
        expected = {"node": {"id": "RmFjdGlvbjoy", "name": "Galactic Empire"}}
        result = await graphql(StarWarsSchema, query)
>       assert result == (expected, None)
E       AssertionError: assert ExecutionResult(data={'node': None}, errors=[GraphQLError('Support for returning GraphQLObjectType from resolve_type was removed in GraphQL-core 3.2, please return type name instead.', locations=[SourceLocation(line=3, column=9)], path=['node'])]) == ({'node': {'id': 'RmFjdGlvbjoy', 'name': 'Galactic Empire'}}, None)
E         +ExecutionResult(data={'node': None}, errors=[GraphQLError('Support for returning GraphQLObjectType from resolve_type was removed in GraphQL-core 3.2, please return type name instead.', locations=[SourceLocation(line=3, column=9)], path=['node'])])
E         -({'node': {'id': 'RmFjdGlvbjoy', 'name': 'Galactic Empire'}}, None)
E         Full diff:
E         - ({'node': {'id': 'RmFjdGlvbjoy', 'name': 'Galactic Empire'}}, None)
E         + ExecutionResult(data={'node': None}, errors=[GraphQLError('Support for returning GraphQLObjectType from resolve_type was removed in GraphQL-core 3.2, please return type name instead.', locations=[SourceLocation(line=3, column=9)], path=['node'])])

tests/test_star_wars_object_identification.py:73: AssertionError
________________________ test_correctly_refetches_xwing ________________________

    @mark.asyncio
    async def test_correctly_refetches_xwing():
        query = """
          query XWingRefetchQuery {
            node(id: "U2hpcDox") {
              id
              ... on Ship {
                name
              }
            }
          }
        """
        expected = {"node": {"id": "U2hpcDox", "name": "X-Wing"}}
        result = await graphql(StarWarsSchema, query)
>       assert result == (expected, None)
E       AssertionError: assert ExecutionResult(data={'node': None}, errors=[GraphQLError('Support for returning GraphQLObjectType from resolve_type was removed in GraphQL-core 3.2, please return type name instead.', locations=[SourceLocation(line=3, column=9)], path=['node'])]) == ({'node': {'id': 'U2hpcDox', 'name': 'X-Wing'}}, None)
E         +ExecutionResult(data={'node': None}, errors=[GraphQLError('Support for returning GraphQLObjectType from resolve_type was removed in GraphQL-core 3.2, please return type name instead.', locations=[SourceLocation(line=3, column=9)], path=['node'])])
E         -({'node': {'id': 'U2hpcDox', 'name': 'X-Wing'}}, None)
E         Full diff:
E         - ({'node': {'id': 'U2hpcDox', 'name': 'X-Wing'}}, None)
E         + ExecutionResult(data={'node': None}, errors=[GraphQLError('Support for returning GraphQLObjectType from resolve_type was removed in GraphQL-core 3.2, please return type name instead.', locations=[SourceLocation(line=3, column=9)], path=['node'])])

tests/test_star_wars_object_identification.py:90: AssertionError
```

**Comment by Cito** [comment]
Yes, this is not compatible with core 3.2.

I plan to create a patch release that restricts core to 3.1, and then an update that also supports core 3.2.

Unfortunately, currently we cannot deploy to PyPI because travis-ci.org stopped working. We need @syrusakbary to help with this, I already contacted him.

**Comment by Cito** [comment]
Good news, @syrusakbary already responded, so I can move forward here this week.

**Comment by Cito** [comment]
Version 3.1.1 has now been released, compatible with GraphQL-core 3.1.x, and based on GraphQL-relay-js 0.7.0.

**Comment by markedwards** [comment]
Any estimate on compatibility with core 3.2?

**Comment by Cito** [comment]
Version 3.2.0 has now been released, compatible with GraphQL-core 3.2.x, based on the current GraphQL-relay-js 0.10.0.
